### ISSUE TITLE
[Animated] Regression runtime error if passing undefined to Animated.View style

### ISSUE THREAD

**Issue by Kudo** [necolas/react-native-web#2523]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

If passing `undefined` to Animated.View style, there will be a runtime error. This issue does not happen on react-native-web 0.18. I think this should be a regression issue.

Example code:

```jsx
function App() {
  return (
    <View style={{ flex: 1 }}>
      <Animated.View style={{ height: 64, transform: undefined }}>
        <View style={{ width: 50, height: 50, backgroundColor: "green" }} />
      </Animated.View>
    </View>
  );
}
```

Error stacktrace:

```
AnimatedTransform.js:62 Uncaught TypeError: Cannot read properties of undefined (reading 'forEach')
    at AnimatedTransform.__attach (AnimatedTransform.js:62:1)
    at AnimatedTransform.__addChild (AnimatedWithChildren.js:34:1)
    at AnimatedStyle.__attach (AnimatedStyle.js:86:1)
    at AnimatedStyle.__addChild (AnimatedWithChildren.js:34:1)
    at AnimatedProps.__attach (AnimatedProps.js:63:1)
    at new AnimatedProps (AnimatedProps.js:29:1)
    at useAnimatedProps.js:29:1
    at mountMemo (react-dom.development.js:17225:1)
    at Object.useMemo (react-dom.development.js:17670:1)
    at useMemo (react.development.js:1650:1)
```

### Expected behavior

the undefined style should not raise runtime error.

### Steps to reproduce

I have a minimal reproducible example created from create-react-app

```sh
$ git clone https://github.com/Kudo/repro-rn-web-animated-transform.git
$ cd repro-rn-web-animated-transform
$ npm i
$ npm start
```

### Test case

https://github.com/Kudo/repro-rn-web-animated-transform/

**Comment by necolas** [comment]
In the future, please create test cases using the link provided in the issue template. It saves me a lot of time and inconvenience. Creating a separate repo isn't helpful

**Comment by Kudo** [comment]
sorry for the inconvenience. i did follow the issue template. in the "Test case" field, does it require a codesandbox rather than a separate repo?

```
  - type: input
    attributes:
      label: Test case
      description: Please provide a link to a reduced test case that reproduces the issue.
      placeholder: "https://codesandbox.io/s/6lx6ql1w5r"
    validations:
      required: true
```

**Comment by necolas** [comment]
It's ok. There's a test case for the same issue in https://github.com/necolas/react-native-web/issues/2528

**Comment by Kudo** [comment]
cool, using codesandbox makes the repro much easier. will try to do that next time. thanks!

**Comment by dan-doyon-endear** [comment]
I don't do PRs but here is a patch that works for us.

diff --git a/node_modules/react-native-web/dist/vendor/react-native/Animated/nodes/AnimatedTransform.js b/node_modules/react-native-web/dist/vendor/react-native/Animated/nodes/AnimatedTransform.js
index 66072d9..b92ada1 100644
--- a/node_modules/react-native-web/dist/vendor/react-native/Animated/nodes/AnimatedTransform.js
+++ b/node_modules/react-native-web/dist/vendor/react-native/Animated/nodes/AnimatedTransform.js
@@ -16,7 +16,7 @@ import NativeAnimatedHelper from '../NativeAnimatedHelper';
 class AnimatedTransform extends AnimatedWithChildren {
   constructor(transforms) {
     super();
-    this._transforms = transforms;
+    this._transforms = transforms || [];
   }
   __makeNative() {
     this._transforms.forEach(transform => {

**Comment by Kudo** [comment]
much appreciated for the fix and releasing 0.19.6, thanks @necolas!
